packages/core/src/credential/Credential.ts

```diff
@@ -6,19 +6,19 @@
  */
 
 /**
- * In KILT, an [[AttestedClaim]] is a **credential**, which a Claimer can store locally and share with Verifiers as they wish.
+ * In KILT, an [[Credential]] is a **credential**, which a Claimer can store locally and share with Verifiers as they wish.
```

Comment:
```suggestion
 * In KILT, a [[Credential]] is a **credential**, which a Claimer can store locally and share with Verifiers as they wish.
```

Reply:
I think we should rephrase this, as saying that a credential is a credential sounds a bit funny.

Reply:
17564b9048707f12bf5a5d1161b3b1bfb60c8ca8

Reply:
664679c91ea360ad50aa13127ab8c99781c6fb5d for the rewording